src/pages/WorkPage/CommentSection/CommentInput/SendButton/index.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from "./index.module.css";
+
+import type React from "react";
+
+interface SendButtonProps {
+  onClick?: () => void;
```

Comment:
オプションである必要はないのでこちらは必須項目に


```suggestion
  onClick: () => void;
```